client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/preview-toolbar.tsx

```diff
@@ -30,6 +41,41 @@ const DesignPickerPreviewToolbar = ( {
 		phone: { title: translate( 'Phone' ), icon: phone, iconSize: 24 },
 	} );
 
+	const [ stickyStyle, setStickyStyle ] = useState( {} );
+	const headerRef = useRef< HTMLDivElement >( null );
+	const headerContentRef = useRef< HTMLDivElement >( null );
+
+	useEffect( () => {
+		if ( ! isSticky ) {
+			setStickyStyle( {} );
+			return noop;
+		}
+
+		const handleSticky = () => {
+			if ( ! headerRef.current || ! headerContentRef.current ) {
+				return;
+			}
+
+			const { left } = headerRef.current.getBoundingClientRect();
+
+			setStickyStyle( {
+				position: 'fixed',
+				top: '109px',
```

Comment:
Can we add a comment about what this magic number refers to?

Reply:
Okay